app/src/layouts/calendar/components/agenda.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+	<div class="agenda-wrapper">
+		<div class="agenda" v-if="Object.keys(dayList).length > 0">
+			<div class="day" v-for="(day, i) in dayList" :key="i">
+				<div class="date">Activity on {{ dateOfIndex[i] }}</div>
+				<div class="events">
+					<div class="event" v-for="(event, j) in day" :key="j">
+						<div class="line">
+							<div class="dot" :style="getDotColor(event)"></div>
+						</div>
+						<div class="content" @click="onCLick(event)">
+							<span class="time" v-if="event.__time__">{{ event.__time__.substr(0, 5) }}</span>
+							<span class="title">{{ event.title }}</span>
+						</div>
+					</div>
+					<div class="event add">
+						<div class="line"><div class="dot"></div></div>
```

Comment:
prefer self-closing for empty elements

```suggestion
						<div class="line"><div class="dot"/></div>
```